From d3abf289586f9287bcd94efd19b4ec67faf53b1d Mon Sep 17 00:00:00 2001
From: Ranjan Dutta <ranjan.dutta@intel.com>
Date: Fri, 12 Aug 2022 17:03:55 +0800
Subject: [PATCH 17/46] Revert "drm/msm: return an error pointer in
 msm_gem_prime_get_sg_table()"

This reverts commit ab888b1a9a6df78ccc1c9773d13d7d5e809f9a16.
---
 drivers/gpu/drm/msm/msm_gem_prime.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/msm/msm_gem_prime.c b/drivers/gpu/drm/msm/msm_gem_prime.c
index 705a834ba1e6..d7c8948427fe 100644
--- a/drivers/gpu/drm/msm/msm_gem_prime.c
+++ b/drivers/gpu/drm/msm/msm_gem_prime.c
@@ -17,7 +17,7 @@ struct sg_table *msm_gem_prime_get_sg_table(struct drm_gem_object *obj)
 	int npages = obj->size >> PAGE_SHIFT;
 
 	if (WARN_ON(!msm_obj->pages))  /* should have already pinned! */
-		return ERR_PTR(-ENOMEM);
+		return NULL;
 
 	return drm_prime_pages_to_sg(msm_obj->pages, npages);
 }
-- 
2.27.0

